lib/ios/AirMaps/AIRMapManager.m

```diff
@@ -144,7 +145,7 @@ - (UIView *)view
             MKMapCamera *mapCamera = [[mapView camera] copy];
              [mapCamera setPitch:angle];
              [mapCamera setHeading:bearing];
-
+             [mapCamera setZoom:zoom];
```

Comment:
I don't think this will work - MKMapCamera does not have a zoom property - it is called "altitude": https://developer.apple.com/documentation/mapkit/mkmapcamera and it is in meters.

The problem here then is that the zoom level in Google Maps is a completely different measurement than the altitude on iOS. My suggestion would be to initially implement both properties separately: A user wanting to set the height of the camera would need to provide a property "zoom" on Google Maps and an property "altitude" on iOS.

Later, we night want to add the necessary calculations ourselves to convert between the two.

Reply:
Those calculations would be very handy to have.